GliaWidgets/Sources/EntryWidget/EntryWidget.swift

```diff
@@ -42,6 +42,7 @@ public final class EntryWidget: NSObject {
         observeSecureUnreadMessageCount()
 
         Publishers.CombineLatest(environment.queuesMonitor.$state, $unreadSecureMessageCount)
+            .receive(on: DispatchQueue.main)
```

Comment:
@rasmustautsglia can you add comment here about using here injected combine scheduler?

Reply:
Added [3126e69](https://github.com/salemove/ios-sdk-widgets/pull/1236/commits/3126e69b109b295c1c17755ab5c5e6705e8c76f1)